acmw-db-app/styles/components/AttendanceForm.module.css

```diff
@@ -15,4 +15,9 @@
   text-decoration: none;
   border-radius: 10px;
   overflow: auto;
+}
```

Comment:
I'm not sure if we need this since it seems like this is linked to `SignIn.module.css` instead of `AttendanceForm.module.css` now. I'm sure we'll rearrange how all the styling works though.